packages/ember-application/lib/system/resolver.js

```diff
@@ -117,11 +117,9 @@ export default EmberObject.extend({
   init() {
     this._parseNameCache = dictionary(null);
   },
+
   normalize(fullName) {
-    let [
-      type,
-      name
-    ] = fullName.split(':', 2);
```

Comment:
last time I check second arguments slows down split, (I think thats why its not widely used in code)